packages/core/src/tracing/trace.ts

```diff
@@ -203,6 +203,53 @@ export function getActiveSpan(): Span | undefined {
   return getCurrentHub().getScope().getSpan();
 }
 
+/**
+ * Continue a trace from a sentryTrace & baggage header.
+ * This method takes a sentryTrace, baggage, and an optional request,
+ * and will continue tracing from the context provided.
+ * It takes a callback that receives a transactionContext that may be used for `startTransaction` or `startSpan`.
+ */
+export function continueTrace<V>(
```

Comment:
l: I'd add meta tags here for the backend->frontend use case.

```suggestion
/**
 * Continue a trace from a `sentry-trace` and `baggage` values.
 * These values can be obtained from incoming request headers or, in the browser, from `<meta name="sentry-trace">` and `<meta name="baggage">` HTML tags.
 * 
 * This function takes a `sentry-trace`, `baggage`, and optionally, a `request` object.
 * It will continue tracing from the context provided.
 * 
 * It takes a callback that receives a transactionContext that may be used for `startTransaction` or `startSpan`.
 */
export function continueTrace<V>(
```

Reply:
I updated the comment to reflect this 👍